dt-impact-tracer/src/edu/washington/cs/dt/impact/Main/ResultsParser.java

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
Please delete this clas, dependentTestGenerator and Project789 if they are no longer needed.